src/mono/System.Private.CoreLib/src/System/Reflection/RuntimeMethodInfo.Mono.cs

```diff
@@ -869,19 +869,28 @@ private void InvokeClassConstructor()
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        internal object InvokeCtorWorker(BindingFlags invokeAttr, Span<object?> arguments)
+        internal object InvokeCtorWorker(BindingFlags invokeAttr, in Span<object?> arguments)
         {
             bool wrapExceptions = (invokeAttr & BindingFlags.DoNotWrapExceptions) == 0;
             return InternalInvoke(null, arguments, wrapExceptions)!;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal object InvokeArrayCtorWorker(in Span<int> arguments)
```

Comment:
The general guidance is to avoid passing `Span` as `in`. I understand that you need it for the FCall, but the rest of the code should follow the general guidance.

Reply:
For example, local variable passed as `in` makes the local address-taken that tends to produce less efficient code. Using `in` for Span is a very complex trade-off.

Reply:
Grr. Boo.